src/sentry/utils/performance_issues/__init__.py

```diff
@@ -0,0 +1,3 @@
+__all__ = ("detect_performance_issue",)
+
+from .performance_detection import detect_performance_issue
```

Comment:
this makes it very easy to have an import cycle -- I would prefer an explicit import from the module that houses this

we're going to be tackling these holistically soon to address some import time / isolation issues so it might be best to start now with keeping `__init__.py` empty :shrug: 

feel free to ignore though, we'll handle it with a script and tooling later

Reply:
Nah, works for me, takes two seconds to change it 👍